test/README.md

```diff
@@ -0,0 +1,95 @@
+## How To Test Verdaccio
+
+Welcome to the testing folder at Verdaccio. This document aims to help you to go throug how Verdaccio show be tested.
```

Comment:
throug => through
show => can (or do you mean should?)

Reply:
can :-) is perfect